### PR TITLE
Set up dependency management

### PR DESCRIPTION
**Pull request by sven1103**
This PR configures the dependency management

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,83 +1,110 @@
...
+        <checksumPolicy>fail</checksumPolicy>
+        <enabled>true</enabled>
+        <updatePolicy>always</updatePolicy>
+      </releases>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+      <url>https://qbic-repo.qbic.uni-tuebingen.de/repository/maven-releases</url>
+    </repository>
+  </repositories>
```
Comment (jenniferboedker) [inline pom.xml:78]: Shouldn't those two repos be first in order to reduce compile time?

File: webapp/pom.xml [hunk 2]
```diff
@@ -70,21 +70,13 @@
                 <artifactId>domain</artifactId>
                 <version>1.0-SNAPSHOT</version>
             </dependency>
-            <dependency>
-                <groupId>com.vaadin</groupId>
-                <artifactId>vaadin-bom</artifactId>
-                <version>${vaadin.version}</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
```
Comment (jenniferboedker) [inline webapp/pom.xml:79]: why did you move the vaadin dependency? I would expect it on the app layer and not in the webapp since only here there should be UI stuff.

**Review by jenniferboedker — COMMENTED** [review]
Thanks for reorganizing, I have added some comments.

File: webapp/pom.xml (hunk 2 above)
Comment (sven1103) [reply webapp/pom.xml]: true, i have refactored it

File: pom.xml (hunk 1 above)
Comment (sven1103) [reply pom.xml:78]: no, exactly the oposite. We dont want to search third party dependencies in the Vaadin or our repo. This will lead to increased latency due to the empty requests to these repos, and we know a-priori that most of the stuff is in Maven central :)

**Review by jenniferboedker — APPROVED** [review]
